doc/releases/release-notes-3.3.rst

```diff
@@ -757,6 +757,16 @@ Devicetree
       * :dtcompatible: `st,stm32-ethernet`: now allows ``local-mac-address`` and
          ``zephyr,random-mac-address`` properties.
 
+* STM32F1xx HAL
```

Comment:
This does not fit the previous section 'Bindings'.
A new section 'Others' should be added. In this section, add a subsection `STM32F1 SoCs`, then your changes